Makefile

```diff
@@ -172,6 +177,55 @@ docker-push:
 	docker push ${DELETION_DEFENDER_IMG}
 	docker push ${UNMANAGED_DETECTOR_IMG}
 
+__tooling-image:
+	docker buildx build build/tooling \
+		--platform="$(PLATFORM)" \
+		--output=$(OUTPUT_TYPE) \
+		-t kcc-tooling
+
+__controller-gen: __tooling-image
+CONTROLLER_GEN=docker run --rm -v $(shell pwd):/wkdir kcc-tooling controller-gen
+
+.PHONY: rename-crds
+rename-crds:
+	@echo "Renaming generated CRDs..."
+	@cd config/crds/tmp && \
+	for file in *.yaml; do \
+		if [ "$$file" != "kustomization.yaml" ]; then \
+			base_name=$$(echo "$$file" | sed 's/apiextensions.k8s.io_v1_customresourcedefinition_//; s/.yaml//'); \
+			domain=$$(echo "$$base_name" | cut -d'_' -f1); \
+			resource=$$(echo "$$base_name" | cut -d'_' -f2-); \
+			new_name="apiextensions.k8s.io_v1_customresourcedefinition_$${resource}.$${domain}.yaml"; \
+			mv "$$file" "$$new_name"; \
+			echo "Renamed $$file to $$new_name"; \
+		fi \
+	done
+
+.PHONY: apis-manifests
+apis-manifests: __controller-gen
+	# Clean previous outputs
+	rm -rf $(CRD_OUTPUT_TMP)
+
+	mkdir -p $(CRD_OUTPUT_TMP)
+
+	# Generate CRD manifests into a temporary directory
+	$(CONTROLLER_GEN) crd:allowDangerousTypes=true paths="./apis/resources/logging/..." output:crd:artifacts:config=$(CRD_OUTPUT_TMP)
+
+	# Copy Kustomization template into the temporary CRD directory
+	cp config/crds/kustomization.yaml $(CRD_OUTPUT_TMP)/kustomization.yaml
```

Comment:
There are a few deltas (that I know of).  refs are tricky and I think they need custom code, because I don't think controller-runtime / controller-tools lets us customize the OpenAPI as much as we are doing today.  I took a stab at it in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1605

Once we're writing go code anyway, we might prefer more go tools to kustomize ... TBD